plugins/Destination.validator.mapcss

```diff
@@ -0,0 +1,44 @@
+/*#########################################################################
+##                                                                       ##
+## Copyrights Frédéric Rodrigo 2018                                      ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+#########################################################################*/
+
+meta {
+    title: "Osmose-QA – Destination";
+    description: "Check for invalid characters in destination tags";
+    author: "Andrea Decorte";
+    min-josm-version: 14481;
+    -osmoseTags: list("tag", "highway", "waterway");
+}
+meta[lang=en] { /* lang=en, unused, only to use tr() to catch string for translation */
+    description: tr("Check for invalid characters in destination tags");
+}
+meta[lang=fr] {
+    description: "Vérification des tags destination";
+}
+
+/* ways with "|" in destination tag excluding destination:lanes:*  */
+way[highway][destination][destination*="|"],
+way[highway][/^destination:/][!/^destination:lanes/][/^destination:/=~/\|/],
+way[waterway][destination][destination*="|"],
+way[waterway][/^destination:/][!/^destination:lanes/][/^destination:/=~/\|/]
+{
+    -osmoseItemClassLevel: "3320/33200:0/3";
+    throwError: tr("{0} contains a pipe character", "{1.tag}");
+    group: tr("Pipe characters should not be used in destination tag, only in destination:lanes");
+    suggestAlternative: tr("In case of multiple values, use instead a semicolon to separate values");
```

Comment:
Can you add some tests with `assertMatch:` and `assertNoMatch`.

You can also add `fixAdd` with `;`.

Reply:
Thanks, adding those, only tested on a country extract.
I am adding a fixRemove since this is about a wrong character

Reply:
It is `fixChange`, since you want to suggest change on the value of a tag.

Reply:
All right, I wanted to use that but I didn't see it in the converter (https://github.com/osm-fr/osmose-backend/blob/47a461a2242efb8b80999ed21566c3d463d5a86a/mapcss/mapcss2osmose.py#L221) nor used in the existing files, I will give it a try

Reply:
It is fixAdd, like this: https://github.com/osm-fr/osmose-backend/blob/master/plugins/Name_MisspelledWordByRegex_Lang_fa.validator.mapcss#L39

Reply:
done, thanks